docs/DeveloperGuide.md

```diff
@@ -384,14 +384,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-**UC02: Add a debt**
+**UC02: List persons**
```

Comment:
Is this for the `list` command? If so, may be better to specifiy `list all persons`.

Also if it is talking about `list` command, there are no invalid inputs because extra parameters for `list` are ignored